test/mpi/util/mtest_common.c

```diff
@@ -315,7 +315,7 @@ void MTestAlloc(size_t size, mtest_mem_type_e type, void **hostbuf, void **devic
 #endif
 
     if (type == MTEST_MEM_TYPE__UNREGISTERED_HOST) {
-        *devicebuf = malloc(size);
+        *devicebuf = calloc(size, 1);
```

Comment:
This change should be split out. I'm not sure it's a good thing to always zero-init using this API. Maybe a flag could be added to the API, or a separate interface.

Reply:
I can add a flag to the API and create a separate commit.

Reply:
Created a separate commit for this.